catalogs/catalog-hive/src/main/java/com/datastrato/gravitino/catalog/hive/HiveCatalogOperations.java

```diff
@@ -157,6 +245,16 @@ public void close() {
       clientPool.close();
       clientPool = null;
     }
+
+    if (checkScheduledExecutor != null) {
+      checkScheduledExecutor.shutdown();
+      checkScheduledExecutor = null;
+    }
+
+    File keyTabFile = new File(String.format("tmp/gravitino-%s-keytab", entity.id()));
```

Comment:
I noticed that you have used `deleteOnExit` to remove the file when JVM terminates, do we need to drop it explicitly here?

Reply:
I dropped them in the close method, too.